benchmark_scripts/bench.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DATADIR=$(pwd)
+
+module load mpi
+
+source quick.rc
```

Comment:
Loading MPI modules and sourcing quick.rc is something we should do in the slurm batch script. There is no guarantee that the current environment is carried over. Best to specify when running the job. The module commands will have to be updated for each compute cluster. We could have an `benchmark-aerosol.sb` batch scripts etc.